### PR TITLE
Displaying tags in "get" 

### PR DESCRIPTION
**Pull request by fcuenca**
I found that the "get" function wasn't displaying the tags associated with each entry. I implemented a fix. Also added some basic build script (using Make) and some unit tests for the change.

### CODE REVIEW

**Comment by fcuenca** [comment]
Tested the output using: `pockyt get -s unread -f '{title} | {tags}'`

Shortcut: `make run`

**Comment by alphapapa** [comment]
Thank you for fixing this, I noticed this recently myself.

It's really nice that you added tests and a Makefile too!  However, you might want to split those into separate PRs, to make this easier to process.  Generally a PR should be about one feature.

**Comment by fcuenca** [comment]
Well, it's actually a single (user) feature. The makefile and unit tests are part of their implementation.

**Comment by alphapapa** [comment]
Yes, but since the project doesn't already have those, they are additional features on the development side.  It would be good to add those separately and then add this feature on top of them.  That way it's easier for the maintainer.  But it's up to him ultimately.

**Comment by arvindch** [comment]
Yes, I would prefer to not introduce a makefile or any testing to this project. Can you strip this PR down to just fix the tags bug? I'll happily merge and push a new version then.

File: pockyt/client.py [hunk 1]
```diff
@@ -165,9 +166,17 @@ def _get(self):
             'title': item.get('resolved_title'),
             'link': item.get('resolved_url'),
             'excerpt': item.get('excerpt'),
-            'tags': item.get('tags'),
+            'tags': self._displayTags(item.get('tags')),
         } for item in items.values())
 
+    def _displayTags(self, item_tags):
```
Comment (arvindch) [inline pockyt/client.py]: Should be `_process_tags` to match style and clarify the verb.

Comment (fcuenca) [reply pockyt/client.py]: Makes sense. Done!

**Comment by fcuenca** [comment]
Well, I write code by writing tests. Removing the test therefore implies removing the fix ;-)

Up to you, of course, if you want to take this fix or not, but I don't ship code without tests. If you're still interested, I can create a new pull request, including the change to adhere to the naming convention.

**Comment by arvindch** [comment]
I appreciate your efforts, I really do. 😄 

But this is just a simple side project I made and would rather keep it that way, with no extra bells and whistles. Plus, adding tests for this feature would mean an obligation to add tests for everything and I don't have the time for it. Sorry!

**Comment by fcuenca** [comment]
No worries. I agree with the "obligation" aspect you point out, although I'd rather call it "positive encouragement" instead 😃 Who knows.... perhaps over time more tests would be added, until they are no longer considered "bells and whistles" but "business as usual."

**Comment by alphapapa** [comment]
I agree, some tests is better than no tests, and having some encourages more to be made.  But, one thing at a time.  :)

**Comment by fcuenca** [comment]
Sure. Whenever you're ready to start adding tests, just let me know. Mine with come with a new feature attached ;-)

**Pull request closed without merging**